docs/docs-content/troubleshooting/enterprise-install.md

```diff
@@ -156,3 +156,44 @@ automatically resolve this issue. If you have self-hosted instances of Palette i
 
    Events:  <none>
    ```
+
+## Scenario - "Too Many Open Files" in Management Cluster
+
+When viewing logs for enterprise management clusters or management clusters using a
+[Private Cloud Gateway](../clusters/pcg/pcg.md), you may encounter a "too many open files" error, which prevents logs
+from tailing after a certain point. To resolve this issue, you must increase the maximum number of file descriptors for
+each node on your cluster.
+
+### Debug Steps
+
+Repeat the following process for each node in your management cluster.
+
+1. Log in to a node in your management cluster.
+
+   ```bash
+   ssh -i <key-name> <user@hostname>
+   ```
+
+2. Switch to `sudo` mode.
+
+   ```bash
+   sudo --login
```

Comment:
You may use another variation in some systems, such as `sudo -s`. As such, it may be worth calling this command out as an example rather than an explicit one.